docs/build-speed.md

```diff
@@ -6,22 +6,21 @@ title: Speeding up your Build phase
 Building your React Native app could be **expensive** and take several minutes of developers time.
 This can be problematic as your project grows and generally in bigger organizations with multiple React Native developers.
 
-With [the New React Native Architecture](new-architecture-app-intro), this problem is becoming more critical
-as you might have to compile some native C++ code in your project with the Android NDK in addition to the native code already necessary for the iOS and Android platforms.
-
 To mitigate this performance hit, this page shares some suggestions on how to **improve your build time**.
 
+:::info
```

Comment:
Does it make sense at all to keep this page, given all the prefab/prebuilt work?

Reply:
Yes as you might still have 3rd party libraries that have C++ code. So there is a native compilation step involved anyway.